samples/bluetooth/enocean/src/main.c

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/enocean.h>
+#include <settings/settings.h>
+#include <dk_buttons_and_leds.h>
+
+static void enocean_button(struct bt_enocean_device *device,
+			   enum bt_enocean_button_action action, u8_t changed,
+			   const u8_t *opt_data, size_t opt_data_len)
+{
+	printk("EnOcean button RX: %02x:%02x:%02x:%02x:%02x:%02x: %s %u %u %u %u\n",
+	       device->addr[0], device->addr[1], device->addr[2],
```

Comment:
It looks like this is printing the device address reverse to how it is usually printed. Is this intentional in this case for this library? Otherwise I would recommend bt_addr_to_str here.
This comment applies to all cases where the address is printed.